src/com/google/guiceberry/junit4/GuiceBerryRule.java

```diff
@@ -16,40 +16,54 @@
 package com.google.guiceberry.junit4;
 
 import com.google.guiceberry.DefaultEnvSelector;
-import com.google.guiceberry.GuiceBerryEnvSelector;
 import com.google.guiceberry.GuiceBerry;
 import com.google.guiceberry.GuiceBerry.GuiceBerryWrapper;
+import com.google.guiceberry.GuiceBerryEnvSelector;
 import com.google.guiceberry.TestDescription;
 import com.google.inject.Module;
-
 import org.junit.rules.MethodRule;
+import org.junit.rules.TestRule;
 import org.junit.runners.model.FrameworkMethod;
 import org.junit.runners.model.Statement;
 
 /**
  * {@link GuiceBerry} adapter for JUnit4 tests.
- * 
+ *
+ * <p>This is the preferred implementation to use, as it does not depend on the client code
+ * supplying the right object for {@code Object target}.
+ *
+ * <p>@see If you are looking for a {@link TestRule}, see {@link GuiceBerryTestRule}.
```

Comment:
The @see serves as context to the paragraph above. Just replace both paragraphs with:

* <p>@see If you are looking for a {@link TestRule}, see {@link GuiceBerryTestRule}. Note that the
*       {@link MethodRule} is the preferred implementation to use for being less error-prone.